grpc-java/grpc-servlet-jakarta/build.gradle

```diff
@@ -6,8 +6,7 @@ plugins {
 dependencies {
     implementation platform(libs.grpc.bom)
     implementation libs.grpc.core
-    compileOnly 'jakarta.servlet:jakarta.servlet-api:5.0.0',
-            'org.apache.tomcat:annotations-api:6.0.53'
```

Comment:
We lost this dep. Was it unused?

Reply:
Yep, unused. Checked with Colin, he noted https://github.com/deephaven/deephaven-core/pull/5299#issuecomment-2218439962